tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -14,7 +16,8 @@ def __init__(self, url: str, username: str = None, password: str = None) -> None
         A Client class instances is created with a server url, username and password.
```

Comment:
Should the client have two different ways to sign in? Would this be confusing for users?

Reply:
indeed, I would also recommend only to have one signin

Reply:
The duplicated sign in was removed in the latest commit